standard/interfaces.md

```diff
@@ -216,22 +218,33 @@ The members of an interface are the members inherited from the base interfaces a
 
 ```ANTLR
 interface_member_declaration
-    : interface_method_declaration
-    | interface_property_declaration
-    | interface_event_declaration
-    | interface_indexer_declaration
+    : constant_declaration
+    | field_declaration
+    | method_declaration
+    | property_declaration
+    | event_declaration
+    | indexer_declaration
+    | static_constructor_declaration
+    | operator_declaration
+    | type_declaration
     ;
 ```
 
-An interface declaration declares zero or more members. The members of an interface shall be methods, properties, events, or indexers. An interface cannot contain constants, fields, operators, instance constructors, finalizers, or types, nor can an interface contain static members of any kind.
+An interface declaration declares zero or more members. The members of an interface shall be constants, fields, methods, properties, events, indexers, operators, constructors, and types, some of which may be instance, others static, as described in the subclauses for each interface member kind.
+
+All interface members implicitly have public access; however, an explicit access modifier ([§7.5.2](basic-concepts.md#752-declared-accessibility)) is permitted.
```

Comment:
This statement doesn't apply to static constructors which become allowed in C# 8 as part of DIMs.